src/main/java/seedu/address/logic/commands/InputGroupParticipationCommand.java

```diff
@@ -22,6 +22,9 @@ public class InputGroupParticipationCommand extends Command {
     public static final String SUCCESS_MSG = "Participation points input successfully!";
     public static final String ATTENDANCE_NOT_MARKED = "Before inputting participation points, "
             + "mark the attendance of the student first!";
+    public static final String PARTICIPATION_POINTS_OUT_OF_RANGE = "Participation points "
+            + "must be between 0 and 1000.";
```

Comment:
Would it be better for code maintainability if we did something like `...0 and " + MAXIMUM_PARTICIPATION_POINTS + ".";` instead?